cerbero/commands/package.py

```diff
@@ -125,7 +125,8 @@ def run(self, config, args):
         else:
             paths = pkg.pack(output_dir, args.no_devel,
                              args.force, args.keep_temp,
-                             not args.no_split)
+                             not args.no_split,
+                             strip_binaries=p.strip)
```

Comment:
I'm not sure if this will fix stripping the tarball packages.
Also now the code inside `if` and `else` seems to be exactly the same

Reply:
You are fixing this correctly right? I will leave it to you :)

Reply:
Right